.gitignore

```diff
@@ -6,5 +6,6 @@ dist/
 zaza.egg-info/
 .coverage
 .vscode/
+.idea/
```

Comment:
This really should be in your global .gitignore.  i.e. we shouldn't have to change ever repository's .gitignore for any particular developers preference on an editor.